.github/workflows/scrape_bills.yml

```diff
@@ -0,0 +1,107 @@
+name: 📑 Scrape bills
+
+on:
+  push:
+    branches:
+      - hcg/gh-actions-events
+
+  schedule:
+    - cron: '15 4 * * 0-4,6'  # At 12:15a, Sunday-Thursday and Saturday
+    - cron: '5,20,35,50 * * * 0-4'  # Every 15 minutes, Sunday-Thursday
+    - cron: '5,20,35,50 0-20 * * 5'  # Every 15 minutes until support window, Friday
+    - cron: '5 21-23 * * 5'  # At 5 past the hour, Friday evening
+    - cron: '35,50 21-23 * * 5'  # At :35 and :50, Friday evening 
+    - cron: '5 0-5 * * 6'  # At 5 past the hour, Saturday morning
+    - cron: '35,50 0-5 * * 6'  # At :35 and :50, Saturday morning
+    - cron: '5,20,35,50 6-23 * * 6'  # Every 15 minutes, Saturday
+
+  workflow_dispatch:
+    inputs:
+      matter_ids:
+        description: Specific bills to scrape? (Comma-separated)
+        type: string
+      window:
+        description: How many days to scrape?
+        type: string
+      rpm:
+        description: How many requests per minute?
+        type: string
+    
+jobs:
+  full_scrape:
+    if: ${{ github.event.schedule == '15 4 * * 0-4,6' }}
+    concurrency: 
+      group: full_scrape
+    uses: ./.github/workflows/scrape.yml
+    with:
+      object_type: bills
+      window: 0
+      rpm: 60
+    secrets:
+      DATABASE_URL: ${{ secrets.DATABASE_URL }}
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      LEGISTAR_API_TOKEN: ${{ secrets.LEGISTAR_API_TOKEN }}
+
+  windowed_scrape:
+    if: ${{ contains(fromJSON('["5,20,35,50 * * * 0-4", "5,20,35,50 0-20 * * 5", "5,20,35,50 6-23 * * 6"]'), github.event.schedule) }}
+    concurrency: 
+      group: windowed_scrape
+    uses: ./.github/workflows/scrape.yml
+    with:
+      object_type: bills
+      window: 0.05
+      rpm: 60
+    secrets:
+      DATABASE_URL: ${{ secrets.DATABASE_URL }}
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      LEGISTAR_API_TOKEN: ${{ secrets.LEGISTAR_API_TOKEN }}
+
+  fast_full_scrape:
+    if: ${{ contains(fromJSON('["5 21-23 * * 5", "5 0-5 * * 6"]'), github.event.schedule) }}
+    concurrency: 
+      group: full_scrape
+    uses: ./.github/workflows/scrape.yml
+    with:
+      object_type: bills
+      window: 0
+      rpm: 0
+    secrets:
+      DATABASE_URL: ${{ secrets.DATABASE_URL }}
```

Comment:
I wonder if we can do a something like [this](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow) to avoid passing secrets explicitly in every job?

```yml
  fast_full_scrape:
    if: ${{ contains(fromJSON('["5 21-23 * * 5", "5 0-5 * * 6"]'), github.event.schedule) }}
    concurrency: 
      group: full_scrape
    uses: ./.github/workflows/scrape.yml
    with:
      object_type: bills
      window: 0
      rpm: 0
    secrets: inherit
```

Reply:
Oh yes, that's exactly what I want. Done!